src/main/java/org/moltimate/moltimatebackend/service/DockingService.java

```diff
@@ -35,38 +43,63 @@
 	@Value( "${openbabel.url}" )
 	private String openBabelURL;
 
+	@Autowired
+	private AlignmentService alignmentService;
+
+	private final String TOPDBCONVERSION = "toPDB";
+	private final String TOPDBQTCONVERSION = "toPDBQT";
+
 	public String dockLigand(DockingRequest request) throws IOException {
 
 		HttpHeaders headers = new HttpHeaders();
 		headers.setContentType(MediaType.MULTIPART_FORM_DATA);
 
-		if (request.getLigand() == null) {
+		if (request.getLigand().isEmpty()) {
```

Comment:
To prevent Null Pointer Exception
```suggestion
		if (request.getLigand() == null || request.getLigand().isEmpty()) {
```